README.md

```diff
@@ -463,6 +463,7 @@ timeout             | Optional open/read timeout in seconds.
 open_timeout        | Optional connection open timeout in seconds.
 publishable_token   | IEX Cloud API publishable token.
 endpoint            | Defaults to `https://cloud.iexapis.com/v1`.
+referrer            | Optional string for HTTP `Referer` header, enables token domain management.
```

Comment:
Link "enables ..." to https://intercom.help/iexcloud/en/articles/3014948-restrict-domains-for-calls-made-with-a-publishable-key.